python/cog/server/redis_queue.py

```diff
@@ -242,9 +243,7 @@ def handle_message(
 
         self.runner.run(**input_obj.dict())
 
-        response["x-experimental-timestamps"] = {
-            "started_at": datetime.datetime.now().isoformat()
-        }
+        response["started_at"] = started_at.isoformat() + "Z"
```

Comment:
Maybe `.isoformat() + "Z"` should go in a function `format_datetime` or something, where can also document why we include the "Z" suffix.